spring-cloud-dataflow-tasklauncher/spring-cloud-dataflow-tasklauncher-sink-rabbit/src/main/java/org/springframework/cloud/stream/app/spring/cloud/dataflow/tasklauncher/sink/rabbit/SpringCloudDataflowTasklauncherSinkRabbitApplication.java

```diff
@@ -18,9 +18,12 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.batch.BatchAutoConfiguration;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.cloud.task.batch.configuration.TaskBatchAutoConfiguration;
 import org.springframework.context.annotation.Import;
 
-@SpringBootApplication
+@SpringBootApplication(exclude = {BatchAutoConfiguration.class, TaskBatchAutoConfiguration.class, DataSourceAutoConfiguration.class})
```

Comment:
This seems really odd that we are having to specify this on the application.

Reply:
I still think we should not need this - but I don't think it is isolated to this location as the cause. We will keep an eye on this when we add ITs for this. @corneil are you planning on adding ITs for this after this PR is merged?